hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieFileIndex.scala

```diff
@@ -108,9 +108,6 @@ case class HoodieFileIndex(spark: SparkSession,
    * @return list of PartitionDirectory containing partition to base files mapping
    */
   override def listFiles(partitionFilters: Seq[Expression], dataFilters: Seq[Expression]): Seq[PartitionDirectory] = {
-    val convertedPartitionFilters =
```

Comment:
This has to be removed since now there partition values are populated from partition path as is, and therefore users will have to query them as they are -- no conversion is needed therefore

Reply:
this also used in org.apache.hudi.MergeOnReadSnapshotRelation#collectFileSplits

Reply:
this means some behavior changes where users still set timestamp keygen for writing, but query the physical partition paths directly without any keygen settings. And also means `convertFilterForTimestampKeyGenerator()` won't be needed?

Reply:
MOR still does the conversion w/in the Relation itself

Reply:
@xushiyan this reverts us back to behavior we had in 0.10 for COW -- original columns will be overridden with partition values by Spark